website/content/en/status/report-2023-01-2023-03/cloud-init.adoc

```diff
@@ -0,0 +1,27 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst]
```

Comment:
While you fix the other things, it would be helpful if you could empty the last pair of brackets of each line so that I do not need to update https://reviews.freebsd.org/D39571. Thanks.